pkg/probe/probe_registration.go

```diff
@@ -109,10 +110,22 @@ func (dMetadata *DiscoveryMetadata) SetFullRediscoveryIntervalSeconds(fullDiscov
 
 // IActionPolicyProvider provides the policies for action types supported for
 // different entity types by the probe
+// Returns action policies defined by the probe. Each action policy is associated with an
+// EntityType and represents a list of actions with their modes (Disabled, Recommend, etc.).
 type IActionPolicyProvider interface {
 	GetActionPolicy() []*proto.ActionPolicyDTO
 }
 
+// The default action policy metadata if the probe does not support action executor.
```

Comment:
Any more detail about the purpose of this DefaultActionPolicyMetadata?
It defines `RecommondOnly` for action `ActionItemDTO_NONE` for entity `EntityDTO_UNKNOWN`.